### ISSUE TITLE
v2 error bag field key name, not working with @error

### ISSUE THREAD

**Issue by tanthammar** [livewire/livewire#1454]
v2 a705946
Using Livewire model hydration based on the `$rules` array does not work with blade `@error` directive.
```php
    public $user;

    protected $rules = [
        'user.name' => 'required',
        'user.email' => 'required|email',
    ];

    public function mount(?User $user)
    {
        $this->user = $user;
    }
```

```php
<input type="email" wire:model="user.email" />

@error("user.email")
         <p>{{ $message }}</p>         //   never shows
@enderror

@if ($errors->any())
    <ul>
        @foreach ($errors->all() as $message)
            <li>{{ $message }}</li>                        // error visible
        @endforeach
    </ul>
@endif
```

![image](https://user-images.githubusercontent.com/21239634/91899810-f7d25a00-ec9d-11ea-9f84-b7e74e382204.png)

**Comment by tanthammar** [comment]
Same applies to array fields
```php
    protected $rules = [
        'user.name' => 'required',
        'user.email' => 'required|email',
        'user.friends.*.first_name' => 'required',
        'user.friends.*.last_name' => 'required',
        'user.opening_hours.day' => 'required',
        'user.opening_hours.time' => 'required',
    ];
```

**Comment by calebporzio** [comment]
Can you dump out the error bag? What does it look like?

**Comment by nuernbergerA** [comment]
@tanthammar please try again with the latest master.

And thanks for testing all the stuff :)

**Comment by tanthammar** [comment]
Resolved with 51090b9
